### ISSUE TITLE
Window -> Perspective -> Save Perspective As ... missing

### ISSUE THREAD

**Issue by 2019-05-10** [dbeaver/dbeaver#17089]
Suggesting to reset UI/workspace apparently means losing all configuration of UI (and more? Both pages are suspiciously silent about what exactly each reset entails)
At least for the UI there's a way to store the layout of editors and views in Eclipse, which DBeaver lacks.
For the preferences there's a similar Export/Import feature, that DBeaver likewise lacks.

Given that "Reset this or that" very much sounds like destroying everything user, the lack of backup/restore is surprising.

**Comment by flambchap** [comment]
Hi. We will add the Export/Import feature for settings in the scope of https://github.com/dbeaver/dbeaver/issues/17368.

**Comment by flambchap** [comment]
With regards to `Save Perspective As...`, I'm not sure if it's possible to do it in the Eclipse RCP app. We need to figure that out.

**Comment by flambchap** [comment]
Yes, it seems like we can add the ability to save custom perspectives.
